src/main/java/Deadline.java

```diff
@@ -0,0 +1,34 @@
+/**
+ * Represents a task that is added to the list by a user. A Deadline object contains a description
+ * represented by a String and a done status represented by a boolean. This is all inherited
+ * from superclass Task.
+ * Deadline object contains date that task is to be completed by, represented by a String.
+ * Inherits all methods from superclass Task.
+ */
+public class Deadline extends Task {
+
+    protected String by;
+
+    public Deadline(String description, String by) {
+        super(description);
+        this.by = by;
+    }
+
+    @Override
+    public String saveToFile() {
+        // return format
+        String printStatus;
```

Comment:
Would it be better to be named `status` rather than `printStatus`? `printStatus` sounds more suitable for a method name.